README.md

```diff
@@ -1 +1,143 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🔭 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🎯 School Library<a name="about-project"></a>
+
+> In this project, I am learning about Object Oriented Programming (OOP) in ruby. In this initial step, I implemented the classes to represent students and teachers. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Class**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming Soon... -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+you have those tools in your local machine.
+
+- [ ] RUBY
+
+### Setup
+
+Clone the project.
+
+```bash
+  git clone https://github.com/unorjikingsley/School-Library.git
+```
+
+Go to the project directory.
+
+```bash
+  cd School-Library
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Test
+
+``` test
+No tests here...
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+### Author
+
+👤 **Unorji Kingsley**
+
+- GitHub: [@Kingsley](https://github.com/unorjikingsley)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/unorjikingsley)
+
+👤 **Shaheryar Abid**
+
+- GitHub: [@Shaheryar](https://github.com/Shaheryar0054)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shaheryar-abid/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Adding the optional functionalities]**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/unorjikingsley/School-Library/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## 👋 Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🔭Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse for this opportunity to work on this project and have better understanding of Ruby
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/unorjikingsley/School-Library/blob/enumerable/LICENSE) licensed.
```

Comment:
- [ ] Your MIT license link currently leads to an error page. I have attached a screenshot below. I think this might be because you don't have a license file in your project. So I will suggest that you include an MIT license file in your project. You can use this  [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink:

![image](https://github.com/unorjikingsley/School-Library/assets/102699958/e083f3a2-a86d-4f05-a992-e8aa8b8584d5)